### ISSUE TITLE
No demos listed after 17/03/18

### ISSUE THREAD

**Issue by cybersteel8** [akiver/CSGO-Demos-Manager#309]
CSGO Demos Manager version: 2.13.3

So, CDM is still able to download my demos from my account - it is doing this successfully and putting them into my csgo/replays folder. However, they are not appearing in my list. The most recent demo was at 17/03/18 and I've played about 30 games since. All of these have downloaded, but none are listed. I've tried Clear demos data, Clear VDM files, and removing my account from CDM altogether.

It is this last step that allowed me to see all the demos, however when I add back my account, all the demos after that date disappear again. I have to analyze them while my account is removed for them to remain on the list when I re-add my account.

What's going on here?

**Comment by Cookieseller** [comment]
I'm having no such problem, how many demos do you have overall and could you share a demo after that date?

**Comment by akiver** [comment]
Could you check the dates interval in `Settings => Stats`?

**Comment by cybersteel8** [comment]
Hi, unfortunately I didn't see the notification about the previous message from Cookieseller, sorry about that! I played a game of CS about a week ago and I've loaded up CSGODM now to see if it detected it, and surprisingly it has! It downloaded it, and it appeared in the list for me to select and Analyze. That's unusual since I haven't changed anything or updated the program at all. I don't know what started the problem and I don't know what fixed it either, it sort of just fixed itself?

@akiver for your reference, the From date is 11/30/2011 and To date is today 7/22/2018. I am unsure of what it was last time I opened CSGODM while the problem was occurring unfortunately.
@Cookieseller I could provide a demo from back when the problem was occurring, if it is still important to do. Is it still necessary?

Thanks for your replies, and sorry for not seeing the notification sooner. I wish this didn't fix itself automatically so I could help diagnose what happened further.
